src/modules/commander/calibration_messages.h

```diff
@@ -50,17 +50,17 @@
 
 // The number in the cal started message is used to indicate the version stamp for the current calibration code.
 #define CAL_QGC_STARTED_MSG			"[cal] calibration started: 2 %s"
-#define CAL_QGC_DONE_MSG			"[cal] calibration done: %s"
-#define CAL_QGC_FAILED_MSG			"[cal] calibration failed: %s"
-#define CAL_QGC_WARNING_MSG			"[cal] calibration warning: %s"
-#define CAL_QGC_CANCELLED_MSG			"[cal] calibration cancelled"
+#define CAL_QGC_DONE_MSG			"calibration done: %s"
```

Comment:
I'll check with QGC, but some of these might need to keep [cal].

Reply:
These need to retain [cal] definitely.